disnake/interactions/base.py

```diff
@@ -1289,22 +1357,9 @@ async def edit(
         :class:`InteractionMessage`
             The newly edited message.
         """
-        # if no attachment list was provided but we're uploading new files,
-        # use current attachments as the base
-        if attachments is MISSING and (file or files):
-            attachments = self.attachments
```

Comment:
Ideally this part should be kept to avoid unnecessarily fetching the message in `edit_original_message`

Reply:
https://github.com/DisnakeDev/disnake/pull/469/commits/b5c9805fdc9732d0666a6dcd9d2525776a6708db